Project/core/src/main/java/Controller/GameController.java

```diff
@@ -0,0 +1,103 @@
+package Controller;
+
+import Model.Player;
+import Model.Recruiter;
+import Model.RougeAgent;
+import Model.Game;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameController {
+    // Check for new turn
+    // Think this would be better as view calling it to move from one turn to the
+    // next as needed
+    // Update recruits if odd turn
+    // Activate player
+    // Check win
+
+    private boolean recruiterTurn = true;
+    private Game gameState;
+    private int activePlayer = 0;
+    private Model.Player players[] = new Player[6];
+    private Recruiter recruiter = null;
+    List<RougeAgent> agents = new ArrayList<RougeAgent>();
```

Comment:
It seems like this list is only used in the constrructor, could it be used as a stack variable instead of a class field?